pages/osx/port.md

```diff
@@ -0,0 +1,27 @@
+# port
+
+> Package manager for macOS.
```

Comment:
So `port` is macOS's primary package manager?

Reply:
No, there's also Homebrew which has its brew command already in the TLDR pages. I took that line directly from the brew page. I think port is the more venerable of the two.

Reply:
Ah, ok :smiley_cat:

Reply:
Sorry, forgot I hadn't made that change, just added it now